Code/Engine/Foundation/Configuration/CVar.h

```diff
@@ -120,7 +120,7 @@ class EZ_FOUNDATION_DLL ezCVar : public ezEnumerable<ezCVar>
   /// So it might not be necessary to call this function manually at shutdown.
   static void SaveCVars(); // [tested]
 
-  /// \brief Stores all CVar values into the given file.
+  /// \brief Stores all CVar values into the given file, ignoring the save flag ezCVarFlags::Save.
```

Comment:
The fact that the Save flag is completely ignored by this function is a bit unexpected and easy to miss. It would be great if this was an option to the Save function instead, so that one can choose whether all CVars get saved or only the ones with the flag.

Reply:
Added an option